### PR TITLE
nudge the social share label to appear visually aligned to the icons

### LINKED ISSUE #111: Social icons buttons are not center aligned with "Share on" text
In design arata asa 
CoVid - Ce Ma Fac – Figma 2020-03-28 11 42 41 (1)

iar implementarea arata asa 
Ce trebuie să fac 2020-03-28 11 45 47 (1)

### PR DESCRIPTION
**Pull request by pdamoc**
### What changed?
 - Added a class to the social share label, nudged the label 4px up to visually center on icons
 - Closes  #111

### CODE REVIEW

**Comment by surdu** [comment]
Wouldn't be a better idea to eihter use `verical-align: middle` for all elements in the share component or switch to `display: flex` and `align-items: center` ?

File: src/components/social-share/social-share.scss [hunk 1]
```diff
@@ -5,4 +5,7 @@
   & > * {
     margin: 4px;
```
Comment (moonflare) [inline src/components/social-share/social-share.scss]: Adding `display: flex` here will do the trick.

Comment (moonflare) [reply src/components/social-share/social-share.scss]: Also, we could change that margin to `margin: 7px 4px;` instead to get closer to the design

Comment (pdamoc) [reply src/components/social-share/social-share.scss]: I have implemented the requested changes.

**Pull request merged**